cisco_firepower_management_center/plugin.spec.yaml

```
@@ -333,3 +456,81 @@ actions:
         description: Returns information about the deleted address object
         type: address_object
         required: false
+  check_if_address_in_group:
+    title: Check If Address in Group
+    description: Checks if provided Address Object name or host exists in the Address Group
+    input:
+      group:
+        title: Group
+        description: Name of address group to check
+        type: string
+        required: true
+        example: MaliciousAddressGroup
+      address:
+        title: Address
+        description: Address Object name, or IP, CIDR, or domain name when Enable Search is on
+        type: string
+        required: true
+        example: MaliciousHost
+      enable_search:
+        title: Enable Search
+        description: Boolean to search for contents of Address Objects for IP, CIDR, domain
+        type: boolean
+        required: false
+        default: false
+        example: false
+    output:
+      found:
+        title: Found
+        description: Was address found in group
+        type: boolean
+        required: true
+      address_objects:
+        title: Address Objects
+        description: List of found address objects
+        type: "[]address_object"
+        required: false
+  add_address_to_group:
+    title: Add Address to Group
+    description: Adds an existing address object to a group
+    input:
+      address:
+        title: Address
+        description: Name of address object
+        type: string
+        required: true
+        example: MaliciousHost
+      group:
+        title: Group
+        description: Name of address group to add the address to
+        type: string
+        required: true
+        example: MaliciousAddressGroup
+    output:
+      network_group:
+        title: Network Group
+        description: Returns information about the network group
+        type: network_group
+        required: false
+  remove_address_from_group:
+    title: Remove Address from Group
+    description: Removes an address object from a group
+    input:
+      address:
+        title: Address
+        description: The hostname, an IP address or subnet address expressed in CIDR notation to remove from group
```

Action title says `description: Removes an address object from a group` but this description indicates any hostname or IP is fine. Update to be consistent

```suggestion
        description: The hostname, an IP address or subnet address expressed in CIDR notation to remove from group
```